controls/radrichtextbox/document-elements/table/api.md

```diff
@@ -0,0 +1,69 @@
+--- 
```

Comment:
Can we add some examples in this topic? The members are exposed by different classes and that might confuse the customers.